src/pages/SendSwapRequest/SendSwapRequest.tsx

```diff
@@ -80,17 +81,26 @@ const SendSwap = ({ location, history }: RouteComponentProps) => {
                 )}
               </Fieldset>
             </Grid>
-            <Grid item={true} xs={12}>
+            <Grid item={true} xs={12} md={6}>
               <Fieldset legend={"To"}>
                 <TextField
-                  value={peer}
-                  onChange={event => setPeer(event.target.value)}
+                  value={peerId}
+                  onChange={event => setPeerId(event.target.value)}
                   label={"Peer"}
                   helperText={"Peer ID"}
                   data-cy="peer-input"
                 />
               </Fieldset>
             </Grid>
+            <Grid item={true} xs={12} md={6}>
+              <TextField
+                value={addressHint}
+                onChange={event => setAddressHint(event.target.value)}
+                label={"Peer Address Hint"}
+                helperText={"Address in multiaddress format"}
```

Comment:
:mega: Should be aligned with the helperText from the other component.

You could extract your own component from this one :)

Reply:
I just saw that it is actually duplicated 3 times, with different hints and `data-cy` attributes. Please extract a component from that one.